site/profile/manifests/foreman_aio.pp

```diff
@@ -60,6 +60,15 @@
     },
   }
 
+  class { 'hiera':
+    hierarchy      => [
+      'nodes/%{::trusted.certname}',
```

Comment:
This seems odd to me:

1. single quote == no interpolation
2. `%{...}` implies interpolation

What am I missing here?